airo-camera-toolkit/airo_camera_toolkit/point_clouds/operations.py

```diff
@@ -61,3 +62,17 @@ def crop_point_cloud(
     """
     crop_mask = generate_point_cloud_crop_mask(point_cloud, bounding_box)
     return filter_point_cloud(point_cloud, crop_mask.nonzero())
+
+
+def transform_point_cloud(point_cloud: PointCloud, frame_transformation: HomogeneousMatrixType) -> PointCloud:
```

Comment:
So frame transformation is the `X_A_B` transform where B is the current frame in which the pointcloud is expressed and A is the new frame in which you want to express it?

Reply:
Yes. For example:

- You capture point clouds from a camera at a fixed location with extrinsics `X_W_C`.
- You transform the point cloud to the world frame using `transform_point_cloud(point_cloud, X_W_C)`.